.github/workflows/pr-ci.yml

```diff
@@ -1,85 +1,158 @@
 name: CI build and push (PR)
 
 concurrency:
-  group: pr-${{ github.event.number }}
+  group: ci-${{ github.run_id }}
   cancel-in-progress: true
 
 on:
-  pull_request_target:
+  issue_comment:
     types:
-      - opened
-      - reopened
-      - synchronize
-      - labeled
-      - unlabeled
-    branches:
-      - main
-      - v[0-9]+
-      - v[0-9]+.[0-9]+
-      - cryostat-v[0-9]+.[0-9]+
+      - created
 
 jobs:
+  check-before-build:
```

Comment:
Excluding the `needs-triage` case and unexpected issue, looks like this job will also complete successfully. This will cause the other jobs (i.e. build, test, code-analysis) to run.

Reply:
How about:

1. Create a step "Command permission allowed" that checks if the author is MEMBER or ADMIN  or `safe-to-test` label is applied.
2. Create a next step. If step 1 result is false, print the message.
3. Create another step. If step 1 result is false, exit 1.
4. For step "React to comment", only react "+1" if step 1 returns true. Otherwise, ignore or use "eyes emoji".

Reply:
Once this is done, all other jobs can remove

```
github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/build_test')
```

since they declare needs on `check-before-build` job which already does the check.  Other jobs will be skipped if the `check-before-build`  fails or skips.